shared/heroku_app/main.tf

```diff
@@ -78,34 +83,61 @@ data "aws_ssm_parameter" "slack_deploy_webhook" {
 
 locals {
   config_vars = {
-    APP_URL = "https://${var.domain}"
-
-    # Set environment & ensure app isn't in debug mode:
-    APP_ENV   = "${var.environment}"
-    APP_DEBUG = "false"
-
-    # Configure logging for Heroku drain:
-    APP_LOG = "errorlog"
-
-    # Alongside the Trusted Proxy module, this allows us to use SSL behind
-    # Heroku's load balancer. Heroku strips these headers on incoming traffic
-    # so it's safe to trust all (and we can't know their specific IPs).
-    TRUSTED_PROXY_IP_ADDRESSES = "**"
-
-    # If we're using Redis for this app, tell Laravel to use that for cache & session store.
-    CACHE_DRIVER   = "${var.with_redis ? "redis" : "file"}"
-    SESSION_DRIVER = "${var.with_redis ? "redis" : "file"}"
-
-    # New Relic (if enabled for this app):
-    NEW_RELIC_ENABLED     = "${var.with_newrelic ? "true" : "false"}"
-    NEW_RELIC_APP_NAME    = "${var.with_newrelic ? var.name : ""}"
-    NEW_RELIC_LICENSE_KEY = "${join("", data.aws_ssm_parameter.newrelic_api_key.*.value)}" # optional! <https://git.io/fp2pg>
-    NEW_RELIC_LOG_LEVEL   = "error"
+    express = {
+      APP_URL   = "https://${var.domain}"
+      APP_ENV   = "${var.environment}"
+      NODE_ENV  = "production"
+      LOG_LEVEL = "info"
+
+      # New Relic (if enabled for this app):
+      NEW_RELIC_ENABLED     = "${var.with_newrelic ? "true" : "false"}"
+      NEW_RELIC_APP_NAME    = "${var.with_newrelic ? var.name : ""}"
+      NEW_RELIC_LICENSE_KEY = "${join("", data.aws_ssm_parameter.newrelic_api_key.*.value)}" # optional! <https://git.io/fp2pg>
+      NEW_RELIC_LOG_LEVEL   = "error"
+    }
+
+    laravel = {
+      APP_URL = "https://${var.domain}"
+
+      # Set environment & ensure app isn't in debug mode:
+      APP_ENV   = "${var.environment}"
+      APP_DEBUG = "false"
+
+      # Configure logging for Heroku drain:
+      APP_LOG = "errorlog"
+
+      # Alongside the Trusted Proxy module, this allows us to use SSL behind
+      # Heroku's load balancer. Heroku strips these headers on incoming traffic
+      # so it's safe to trust all (and we can't know their specific IPs).
+      TRUSTED_PROXY_IP_ADDRESSES = "**"
+
+      # If we're using Redis for this app, tell Laravel to use that for cache & session store.
+      CACHE_DRIVER   = "${var.with_redis ? "redis" : "file"}"
```

Comment:
Are there no instances by default where we'll use caching for express apps? Pretty sure *no* since this is to setup GraphQL, but it's only notable diff I've noticed, so asking, just in case. :)

Reply:
We actually do use Redis caching in most (or all?) of our Express apps, but it's not configured with a swappable driver. This `CACHE_DRIVER` is a Laravel-specific setting.